lessons/Data-Science/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,278 @@
+
+@lesson-instruction{
+- With a partner, complete @printable-exercise{comparing-trees.adoc}.
+}
+
+As we increase a tree's efficiency, we can lose accuracy. Conversely, if we focus *too* much on accuracy - making our decision trees bigger and more complex - we risk __overfitting__. Overfitting happens when we teach a model the specific quirks of one particular dataset, preventing it from making reliable predictions about new data.
+
+Decision trees can accurately label and categorize the inputs that they are trained to label and categorize! This tree—which was 100% accurate on the trained dataset—falters when we offer it inputs that are either *unknown*, like the chopstick, or *ambiguous*, like the spork. In both instances, the tree offers up its best guess at a label. __The only way this tree stands a chance of correctly identifying a chopstick or a spork is if we offer it more training!__
+
+As we built our decision trees, we were able to draw on everything we know about every knife, spoon, spork, plate, bowl or mug that we have ever seen. If you were asked to create a decision tree to identify common animals or foods, you could probably do that without much difficulty as well.
+
+Computers, however, build decision trees using only the data we provide... and that data can sometimes be messy. As a result, we may end up with models that are not 100% accurate.
+
+Imagine that you are tasked with building a decision tree that can determine an iris' species (_setosa_, _virginica_, and _versicolor_) based on the varying plant measurements. It would be far more challenging to build this tree than the one we just made that classifies common tableware!
+
+@centered-image{images/iris-data.png}
+
+=== Synthesize
+
+@QandA{
+
+@Q{How is playing 20 questions to figure out a mystery item _similar_ to the predicting and recommending that a decision tree enables?}
+@A{Decision trees, like 20 Questions participants, need to avoid questions that don't provide information. Decision trees, like 20 Questions participants, sometimes fail to identify an object correctly. Familiarity with the dataset improves the accuracy of both a decision tree and a 20 Questions participant.}
+
+@Q{How is it different?}
+@A{When we play 20 Questions, we are trying to identify a single object. Decision trees are useful for categorizing a variety of objects, or predicting outcomes in a variety of different scenarios.}
+}
+
+
+
+== Decision Trees from Training Datasets @duration{25 minutes}
+
+=== Overview
+
+Students build a decision tree that predicts whether different individuals will purchase a video game or not.
+
+=== Launch
+
+We have already built some extremely simple decision trees. We have a sense of the hierarchical structure, flexibility, and versatility of decision trees. We know that decision trees sometimes fail when provided with ambiguous or unknown data. We understand the risk of overfitting our data (building a model that can only succeed on narrow set of data).
```

Comment:
What's the youngest age group this lesson is for? Do you expect them to know what "hierarchical" means?

Reply:
blergh, Flannery thought this paragraph sucked too! I will make it better.